tests/test_dfa.py

```diff
@@ -683,23 +683,13 @@ def test_isdisjoint(self):
         """Should test if two DFAs are disjoint"""
         # This DFA accepts all words which contain at least
         # three occurrences of 1
-        A = DFA(
-            states={'q0', 'q1', 'q2', 'q3'},
-            input_symbols={'0', '1'},
-            transitions={
-                'q0': {'0': 'q0', '1': 'q1'},
-                'q1': {'0': 'q1', '1': 'q2'},
-                'q2': {'0': 'q2', '1': 'q3'},
-                'q3': {'0': 'q3', '1': 'q3'}
-            },
-            initial_state='q0',
-            final_states={'q3'}
-        )
+        input_symbols = {'0', '1'}
+        A = DFA.contains_subsequence(input_symbols, '111')
```

Comment:
@Tagl Similar with the past PR, please use lowercase variable names instead of `A`, `B`, `C`, etc.

Reply:
Everything I touched in this PR has been updated